ballerina/types.bal

```diff
@@ -76,6 +78,13 @@ public type ProxyConfig record {|
     string password = "";
 |};
 
+# OAuth2 Client Credentials Grant Configs
+public type OAuth2ClientCredentialsGrantConfig record {|
+    *http:OAuth2ClientCredentialsGrantConfig;
+    # Token URL
+    string tokenUrl = "/oauth/token";
```

Comment:
Is it possible to access tokenURL via this relative path?

Reply:
As per the OpenAPI spec [1] this is doable if token-endpoint is also accessible via API base URL

[1] - [https://swagger.io/docs/specification/authentication/oauth2/](https://swagger.io/docs/specification/authentication/oauth2/)

Reply:
Fixed with following:
- [334978dd9a8af5a894cdea770acf4b63c3c0b1d0](https://github.com/ballerina-platform/module-ballerinax-zuora/pull/4/commits/334978dd9a8af5a894cdea770acf4b63c3c0b1d0)
- [eff6c3b665175be1c2394eaa0e1f1b334c79614c](https://github.com/ballerina-platform/module-ballerinax-zuora/pull/4/commits/eff6c3b665175be1c2394eaa0e1f1b334c79614c)

Reply:
Created following issue [1] to track this

[1] - [https://github.com/ballerina-platform/ballerina-library/issues/6369](https://github.com/ballerina-platform/ballerina-library/issues/6369)